tests/test_flow/test_persistence_gcs.py

```diff
@@ -7,33 +7,36 @@
 These tests are pretty slow -- they take about 60 seconds for me.
 """
 
-import pytest
-
-from pathlib import Path
 import tempfile
+from pathlib import Path
 
 import dask.dataframe as dd
+import pytest
 
+import bionic as bn
+from bionic.exception import CodeVersioningError
+from .fakes import InstrumentedFilesystem
 from ..helpers import (
     df_from_csv_str,
     equal_frame_and_index_content,
     local_wipe_path,
-    gcs_fs_wipe_path,
 )
-from bionic.exception import CodeVersioningError
-
-import bionic as bn
-
 
 # This is detected by pytest and applied to all the tests in this module.
 pytestmark = pytest.mark.needs_gcs
 
 
-@pytest.fixture(scope="function")
-def preset_gcs_builder(gcs_builder):
+@pytest.fixture
+def instrumented_gcs_fs(gcs_fs, use_fake_gcp, make_list):
```

Comment:
What is `use_fake_gcp` used for? Is it still needed?

Reply:
Good catch